homeassistant/components/fritzbox_callmonitor/sensor.py

```diff
@@ -82,10 +79,13 @@ class FritzBoxCallSensor(SensorEntity):
     """Implementation of a Fritz!Box call monitor."""
 
     _attr_icon = ICON_PHONE
+    _attr_has_entity_name = True
+    _attr_translation_key = DOMAIN
+    _attr_device_class = SensorDeviceClass.ENUM
+    _attr_options = [cls.value for cls in CallState]
```

Comment:
I think this change would be better in a separate PR as this PR is only scoped to adding `has_entity_name`